oap-server/server-storage-plugin/storage-iotdb-plugin/README.md

```diff
@@ -0,0 +1,194 @@
+Apache IoTDB - Apache Skywalking适配器
```

Comment:
Chinese document is not recommended to host in SkyWalking codebase.

Our official doc is here, https://github.com/apache/skywalking/blob/master/docs/en/setup/backend/backend-storage.md. And you could read the latest from https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-storage/

Reply:
If IoTDB storage plugin could work well. I will edit official doc later.

Reply:
This design part could land in SkyWalking website as a blog, describing the final status, such as how tables and query/index work.

Reply:
This kind of readme is hard up to date, so, we prefer don't keep them in official repository. If some design principles are important, such as rules of tables, and keys(indices), we should indicate them in the commenta of related codes.

Reply:
Thanks for your suggestion. I will indicate some important design principles in the comments of related codes and delete it from repo in these days. After this project, I will make it as a blog.